### PR TITLE
Add initial documentation for the alertmanager.

### PR DESCRIPTION
**Pull request by brian-brazil**
Add a new section for alerting so it's all together,
and move the alerting rules over there.

### CODE REVIEW

File: content/docs/alerting/rules.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      SUMMARY "High request latency on {{$labels.instance}}"
+      DESCRIPTION "{{$labels.instance}} has a median request latency above 1s (current value: {{$value}})"
+
+### Inspecting alerts during runtime
+To manually inspect which alerts are active (pending or firing), navigate to
+the "Alerts" tab of your Prometheus instance. This will show you the exact
+label sets for which each defined alert is currently active.
+
+For pending and firing alerts, Prometheus also stores synthetic time series of
+the form `ALERTS{alertname="<alert name>", alertstate="pending|firing", <additional alert labels>}`.
```
Comment (discordianfish) [inline content/docs/alerting/rules.md]: Does `alertstate="pending|firing"` work? Or should this be =~?

**Comment by discordianfish** [comment]
LGTM

File: content/docs/alerting/rules.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/alerting/rules.md]: This is a copy of what's already there.

I read this as metasyntax saying alertstate can be either pending or firing, so I think it's okay.

Comment (discordianfish) [inline content/docs/alerting/rules.md]: Hrm.. fine with me although maybe something more explicit would be better:

...the form `ALERTS{alertname="<alert name>", alertstate="<alert state>", <additional alert labels>}`, where alertstate can be `pending` or `firing`.

**Pull request merged**